_posts/2020-11-13-release-2.17.0.md

```diff
@@ -0,0 +1,28 @@
+---
+layout: default
+title: Release v2.17.0
+tags: ["v2.17.0", "changelog"]
+excerpt_separator: <!--more-->
+---
+
+As of 13/11 2020 the new version 2.17.0 is out
+<!--more-->
+
+
+
+## Menas Improvements and Fixes
+
+- [#763] ({{ site.github.issues_url }}/763) UI now displays projected after-conformance schema for datasets.
+- [#1518] ({{ site.github.issues_url }}/1518) Added export and import feature for Schema, Dataset and MappingTable. See the documentation of the REST API for [export]({{ site.baseurl }}/docs/usage/menas-api#export) and [import]({{ site.baseurl }}/docs/usage/menas-api#import).
+
+## Helper scripts Improvements and Fixes
+
+- [#1546] ({{ site.github.issues_url }}/1546) Added version to _Helper scripts_ and this version is now printed with a banner at the beginning of any _Helper script_ run.
+- [#1592] ({{ site.github.issues_url }}/1592) _Helper scripts_ now have a setting to either print a warning or do a non-zero exit in case unknown parameters are passed to them. This config variable `EXIT_ON_UNRECOGNIZED_OPTIONS` can be found in `enceladus_env.sh.template`. Can be set to `"true"` or `"false"`.
+- [#1587] ({{ site.github.issues_url }}/1587) Value for `--null-value` option in helper scripts is properly encapsulated in `"` not to lose white spaces.
```

Comment:
This is not true. `--null-value " alfa "` will still lose the whitespaces. The fix handles empty spaces, not complex values. For complex values use `--null-value "\" alfa \""`

Reply:
Hmm, I think I was testing it, and remember it kept the whitespaces... 🤔 Will double-check tomorrow.

Reply:
You are right (of course, you wrote the feature 😉 ) Isn't it a problem actually?
Going to reword the sentence.